src/components/Modeler.vue

```diff
@@ -385,6 +385,8 @@ export default {
       } else {
         inspectorItems[existingIndex] = config;
       }
+
+      inspectorItems.push(...inspectorItems.splice(inspectorItems.findIndex(item => item.config.name === 'advanced'), 1));
```

Comment:
I find this a little hard to read. Do you think you can split it up into multiple lines or a separate function?

Or maybe this line could be a `sort`, something like:
```js
function isAdvancedAccordion(accordion) {
  return accordion.name === 'advanced';
}

function moveAdvancedAccordionToBottom(accordionA, accordionB) {
  if (isAdvancedAccordion(accordionA)) {
    return 1;
  }

  if (isAdvancedAccordion(accordionB)) {
    return -1;
  }

  return 0;
}

inspectorItems.sort(moveAdvancedAccordionToBottom);
```

Then the sorting behaviour could be unit tested. Actually, the whole `addToInspector` function could be pulled out and unit tested (since it has no reference to `this`)–then the sorting functions would simply be implementation details.

Reply:
I agree with @chipit24 that this could be more readable if broken out. I also agree that it would be great to see a unit test for this.